wwwapp/templatetags/wwwtags.py

```diff
@@ -1,9 +1,21 @@
 from django import template
 from django.utils.html import mark_safe
+from wwwapp.models import WorkshopUserProfile
+
 
 register = template.Library()
 
 
+@register.filter
+def person_status(value):
```

Comment:
This should be done the same way for both people and workshops, and workshops use the Django way of doing this rather than a custom filter :stuck_out_tongue: